tests/test_geoprocessing.py

```diff
@@ -4877,3 +4877,55 @@ def test_iterblocks_bad_raster(self):
         expected_message = 'could not be opened'
         actual_message = str(cm.exception)
         self.assertTrue(expected_message in actual_message, actual_message)
+
+    def test_warp_raster_signedbyte(self):
+        """PGP.geoprocessing: warp raster test."""
+        import pygeoprocessing
+        import pygeoprocessing.testing
+
+        workspace_dir = 'test_workspace'
+        try:
+            os.makedirs(workspace_dir)
+        except OSError:
+            pass
+
+        pixel_a_matrix = numpy.full((5, 5), -1, numpy.int8)
+        nodata_target = -128
+        base_a_path = os.path.join(workspace_dir, 'base_a.tif')
+
+        wgs84_sr = osr.SpatialReference()
+        wgs84_sr.ImportFromEPSG(4326)
+        wgs84_wkt = wgs84_sr.ExportToWkt()
+        gtiff_driver = gdal.GetDriverByName('GTiff')
+        ny, nx = pixel_a_matrix.shape
+        new_raster = gtiff_driver.Create(
+            base_a_path, nx, ny, 1, gdal.GDT_Byte,
+            options=['PIXELTYPE=SIGNEDBYTE'])
+        new_raster.SetProjection(wgs84_wkt)
+        new_raster.SetGeoTransform([1, 1.0, 0.0, 1, 0.0, -1.0])
+        new_band = new_raster.GetRasterBand(1)
+        new_band.SetNoDataValue(nodata_target)
+        new_band.WriteArray(pixel_a_matrix)
+        new_raster.FlushCache()
+        new_band = None
+        new_raster = None
+
+        target_raster_path = os.path.join(workspace_dir, 'target_a.tif')
+        base_a_raster_info = pygeoprocessing.get_raster_info(base_a_path)
+
+        pygeoprocessing.warp_raster(
+            base_a_path, base_a_raster_info['pixel_size'], target_raster_path,
+            'near', target_sr_wkt=wgs84_wkt, n_threads=1,
+            gdal_warp_options=['COPYMETADATA=TRUE'])

```

Comment:
I think our testing concluded that explicitly adding the `['COPYMETADATA=TRUE']` has no effect and is unnecessary. If so, can we remove it here before future us thinks it's important?

Reply:
removed in 1b1f9abae9470570cc276da1e76bfdbcf903db4f